smartsim/experiment.py

```diff
@@ -797,6 +796,56 @@ def reconnect_orchestrator(self, checkpoint: str) -> Orchestrator:
             logger.error(e)
             raise
 
+    def preview(
+        self,
+        output_format: t.Optional[str] = None,
+        output_filename: t.Optional[str] = None,
+        verbosity_level: t.Optional[str] = None,
+    ) -> str:
+        """Preview entity information prior to launch. This method
+        aggregates multiple pieces of information to give users insight
+        into what and how entities will be launched.  Any instance of
+        ``Model``, ``Ensemble`` or ``Orchestrator`` created by the
```

Comment:
May I ask you to use the Oxford comma for consistency with the rest of the documentation?